weave-js/src/components/Panel2/ChildPanelExportReport/ReportSelection.tsx

```diff
@@ -109,7 +109,7 @@ export const ReportSelection = ({
       const foundEntity = entities.result.find(
         (item: EntityOption) => item.name === entityName
       );
-      setSelectedEntity(foundEntity);
+      setSelectedEntity(foundEntity ?? entities.result[0]);
```

Comment:
i just went with selecting the first entity returned by default. an alternative is to disable or hide the report selector until user explicitly chooses an entity